packages/mobile/src/identity/verification.ts

```diff
@@ -333,8 +338,20 @@ function* requestAttestations(
     phoneHash,
     account
   )
+  let isUnselectedRequestValid = unselectedRequest.blockNumber !== 0
+  if (isUnselectedRequestValid) {
+    isUnselectedRequestValid = !(yield call(
+      [attestationsWrapper, attestationsWrapper.isAttestationExpired],
+      unselectedRequest.blockNumber
+    ))
+  }
 
```

Comment:
nit -- this could be simplified
```
isUnselectedRequestValid = unselectedRequest.blockNumber !== 0 
|| !(yield call(
      [attestationsWrapper, attestationsWrapper.isAttestationExpired],
      unselectedRequest.blockNumber
    ))
```
 my understanding is that it shouldn't perform the second part of the OR if the first satisfies to true.

Reply:
It's actually the opposite, it should check the second part if the first one is TRUE. `AND` would do, but I afraid it make a bit harder to understand from the first glance and decided to split it into two `if`s.

Reply:
I see -- in general this is solved by extracting the complexity to a method
```
unselectedRequest.blockNumber !== 0 && !isAttestationExpired(...)
```

no big -- since already merged